rust/src/transformations/polars/make_col/mod.rs

```diff
@@ -0,0 +1,191 @@
+use num::One;
+use opendp_derive::bootstrap;
+use polars::prelude::*;
+use std::collections::BTreeSet;
+
+use crate::core::{Function, MetricSpace, StabilityMap, Transformation};
+use crate::domains::{Context, ExprDomain, ExprMetric, LazyFrameDomain};
+use crate::error::*;
+use crate::traits::DistanceConstant;
+
+#[bootstrap(ffi = false)]
+/// Make a Transformation that returns a col(<column_name>) expression for a Lazy Frame.
+///
+/// # Arguments
+/// * `input_domain` - Domain of the expression to be applied.
+/// * `input_metric` - The metric space under which neighboring LazyFrames are compared.
+///
+/// # Generics
+/// * `M` - Dataset Metric type.
+/// * `C` - Context in which expression is applied.
+///
+pub fn make_col<M, C: Context>(
+    input_domain: ExprDomain<C>,
+    input_metric: M,
+    col_name: String,
+) -> Fallible<Transformation<ExprDomain<C>, ExprDomain<C>, M, M>>
+where
+    M: ExprMetric<C>,
+    M::Distance: DistanceConstant<M::Distance> + One + Clone,
+    (ExprDomain<C>, M): MetricSpace,
+{
+    if input_domain
+        .lazy_frame_domain
+        .column(col_name.as_str())
+        .is_none()
+    {
+        return fallible!(MakeTransformation, "{} is not in dataframe", col_name);
+    }
```

Comment:
We have another helper for this now- `input_domain.lazy_frame_domain.try_column(col_name.as_str())?`

Reply:
I think there should also be a check that the input domain's selected column is `None`.